transport/src/main/java/io/netty/channel/DefaultChannelConfig.java

```diff
@@ -429,4 +424,12 @@ private boolean getPinEventExecutorPerGroup() {
         return pinEventExecutor;
     }
 
+    private ChannelConfig setWakeupOnWrite(boolean wakeupOnWrite) {
+        this.wakeupOnWrite = wakeupOnWrite;
+        return this;
+    }
+
+    private boolean getWakeupOnWrite() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) A getX() method which returns a boolean should be named isX() [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=pmd%3ABooleanGetMethodName)